sv.h

```diff
@@ -1985,13 +1985,24 @@ scalar.
 #  define SvPVbytex_nolen(sv) ((PL_Sv = (sv)), SvPVbyte_nolen(PL_Sv))
 #endif /* __GNU__ */
 
+/*
+=for apidoc Am|bool|SvPVXtrue|SV * sv
+
+Returns a boolean as to whether or not C<sv> contains a PV that is considered
+TRUE.  FALSE is returned if C<sv> doesn't contain a PV, or if the PV it does
+contain is zero length, or consists of just the single character '0'.  Every
+other PV value is considered TRUE.
+
```

Comment:
I don't know if it's standard practice to point out, but this macro may evaluate the `sv` argument more than once. I know that bit me before on the `SvTRUE()` macro, by doing `SvTRUE(POPs)`. It might be worth adding a note here too?

Reply:
I see you've added :)